Added ssl and zlib support

rzuidhof opened the pull request:
Useful for people who want to download plugins within the container

IlyaSemenov commented:
Note that you shouldn't install plugins using the extension manager, as they will be put to ephemeral storage and will be gone on the container restart.

rzuidhof commented:
My experience is that the plugins survive stopping and starting the
container just as well as the data itself. Also restoring is no problem.
Only issue I encountered with your dokuwiki image was that after enabling
nice urls (.htaccess style) uploaded images would not show until I removed
the following lines from the nginx.conf:
#        location ~* \.(js|css|png|jpg|jpeg|gif|ico)$ {
#            expires 31536000s;
#            add_header Pragma "public";
#            add_header Cache-Control "max-age=31536000, public,
must-revalidate, proxy-revalidate";
#            log_not_found off;
#        }

Kind regards,

Richard Zuidhof

On Fri, Nov 25, 2016 at 10:25 AM, Ilya Semenov <notifications@github.com>
wrote:

> Note that you shouldn't install plugins using the extension manager, as
> they will be put to ephemeral storage and will be gone on the container
> restart.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/istepanov/docker-dokuwiki/pull/16#issuecomment-262914263>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AWLso_NP2w_YVtJkOPbKrscOrzHSjvr1ks5rBqlzgaJpZM4K3kAQ>
> .
>

rzuidhof commented:
Sorry, I did not realize I switched to a different Docker image since your
email. I am using vimagick/dokuwiki now, so my comments apply to that
image. There the plugins survive the restart although there is a small
nginx issue.

Kind regards,

Richard Zuidhof

On Tue, Dec 6, 2016 at 10:11 AM, Richard Zuidhof <richard@zuidhof.nl> wrote:

> My experience is that the plugins survive stopping and starting the
> container just as well as the data itself. Also restoring is no problem.
> Only issue I encountered with your dokuwiki image was that after enabling
> nice urls (.htaccess style) uploaded images would not show until I removed
> the following lines from the nginx.conf:
> #        location ~* \.(js|css|png|jpg|jpeg|gif|ico)$ {
> #            expires 31536000s;
> #            add_header Pragma "public";
> #            add_header Cache-Control "max-age=31536000, public,
> must-revalidate, proxy-revalidate";
> #            log_not_found off;
> #        }
>
> Kind regards,
>
> Richard Zuidhof
>
> On Fri, Nov 25, 2016 at 10:25 AM, Ilya Semenov <notifications@github.com>
> wrote:
>
>> Note that you shouldn't install plugins using the extension manager, as
>> they will be put to ephemeral storage and will be gone on the container
>> restart.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/istepanov/docker-dokuwiki/pull/16#issuecomment-262914263>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AWLso_NP2w_YVtJkOPbKrscOrzHSjvr1ks5rBqlzgaJpZM4K3kAQ>
>> .
>>
>
>
